static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -18,4 +20,19 @@ const errorConfig: IssueCategoryConfigMapping = {
   },
 };
 
-export default errorConfig;
+export const titleResourceMap = {
+  [ErrorTitles.CHUNK_LOAD_ERROR]: {
+    resources: {
+      description: t(
+        'ChunkLoadErrors occur when the JavaScript chunks (bundles) that an application is trying to load encounter issues during the loading process. Some common causes are dynamic imports, version mismatching, and code splitting issues. To learn more about how to fix ChunkLoadErrors, check out these resources:'
+      ),
+      links: [
+        {
+          text: t('How to fix ChunkLoadErrors'),
+          link: 'https://sentry.io',
```

Comment:
either lets flag it or wait for the link

Reply:
@scttcper it's ready now